verifier.py

```diff
@@ -330,6 +347,8 @@ def verify():
             print(f"US34-ERR: Family {id} has couples who are large age differences")
 
     for id, individual in individualsDict.items():
+        # US27 Include person's current age when listing individuals
+        print(f"Individual: {id}, name: {individual['name']}, age: {individual['age']}")
```

Comment:
FYI: noah introduced a function `print_individual` in the utils, try using that

Reply:
got it, updated in 10722d7960084879b527035c7b1842cc22542f1b